server/plugin/plg_backend_s3/index.go

```diff
@@ -74,55 +75,55 @@ func (s S3Backend) Init(params map[string]string, app *App) (IBackend, error) {
 	return backend, nil
 }
 
-func (s S3Backend) LoginForm() Form {
+func (s *S3Backend) LoginForm() Form {
 	return Form{
 		Elmnts: []FormElement{
-			FormElement{
+			{
```

Comment:
Please don't reformat the existing code

Reply:
`FormElement` is redundant type from slice, and golint will pop up `redundant type from array, slice, or map composite literal`.

Reply:
This kind of things will one day be part of a different PR, let's not mix things here